packages/widget/src/providers/WidgetProvider/WidgetProvider.tsx

```diff
@@ -17,6 +18,30 @@ export const useWidgetConfig = (): WidgetContextProps =>
 
 let sdkInitialized = false;
 
+const attemptToFindMatchingToAddressInConfig = (
```

Comment:
Here we do the query param toAddress check agains the `toAddress` and `toAddresses` in config values.  We don't have access to the Bookmarks from local storage at this point so we have to do checks against  Boolmark later in the code execute - see the onRehydrateStorage function in createBookmarkStore

Reply:
Looks good. Can we please create `utils.ts` in this folder and move this helper function there? 🙂